examples/tree_1d_dgsem/elixir_shallow_water_quasi_1D_manufactured.jl

```diff
@@ -0,0 +1,59 @@
+using OrdinaryDiffEq
```

Comment:
The name of this elixir does not follow the normal convention. Typically, the convergence test elixirs would be something like `elixir_shallow_water_quasi_1d_source_terms.jl`. Also, we prefer to avoid capital letters in the files names.